docs/DeveloperGuide.md

```diff
@@ -237,70 +237,403 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* *`    | Product Manager                          | flag problematic deliverables in the app                                    | know that these tasks require further attention |
+| `* *`    | Product Manager                          | be notified when my product’s development is close to tagline / past deadline / too slow |                       |
+| `* *`    | Product Manager                          | pin different parts of the project according to their urgency               |                                    |
+| `*`      | Product Manager                          | see the previous products I have handled and their reviews                  |                                    |
+| **Contacts** |
+| `* * *`  | Product Manager                          | add a contact                                                               | store their details for future communication |
+| `* * *`  | Product Manager                          | view my contacts and their relevant details                                 |                                    |
+| `* * *`  | Product Manager                          | edit my contacts                                                            | keep them up-to-date               |
+| `* *`    | Product Manager                          | customise fields of information                                             | keep any data that i want          |
+| `* *`    | Product Manager                          | pin contacts which I frequent                                               | easily search through my long list of contact |
+| `* *`    | Product Manager                          | search for contacts based on a certain field of information                 | easily navigate through different groups |
+| **Meetings** |
+| `* * *`  | Product Manager                          | schedule new meetings with my stakeholders                                  | keep track of them                 |
+| `* * *`  | Product Manager                          | edit existing meeting details with my stakeholders                          | update them accordingly            |
+| `* * *`  | Product Manager                          | delete scheduled meetings with my stakeholders                              | remove outdated or cancelled meetings |
+| `* *`    | Product Manager                          | be notified if I have conflicting meetings before adding new ones           | reschedule the meetings            |
+| `* *`    | Product Manager                          | see a calendar view of my meetings                                          | know my available slots to schedule new meetings |
+| `* *`    | Product Manager                          | toggle between daily, weekly and monthly view                               | have both a macro and micro view of my schedule |
+| **Communication** |
```

Comment:
Did you miss the epic "As an inexperienced or forgetful Product Manager, I can refer to a user guide as I’m using the app so that I am able to use it as intended"? The user stories in this epic are not reflected.

Reply:
Yes, it seems I was careless and glossed over that section. Will include, thank you for pointing out!